walkbox.go

```diff
@@ -93,6 +96,24 @@ func (w *WalkBox) IsAdjacent(otherWalkBox *WalkBox) bool {
 	return false
 }
 
+// Distance calculates the shortest distance from the WalkBox to the given position.
+func (wb *WalkBox) Distance(p *Positionf) float32 {
```

Comment:
This, and other functions from `WalkBox` will not be used out from this type. Hence they should be private.